client/chain_registry/chain_info.go

```diff
@@ -69,10 +69,17 @@ func (c ChainInfo) GetAllRPCEndpoints() (out []string, err error) {
 		var port string
 		if u.Scheme == "https" {
```

Comment:
As you did below for `http`, should we ensure here that someone isn't running `https` on a non standard port (not 443) ?

Reply:
Seems reasonable. Made this change and added some additional tests. Let me know if any additional url schemes should be supported other than `http` and `https`.